src/rgw/rgw_admin.cc

```diff
@@ -4491,7 +4491,8 @@ int main(int argc, const char **argv)
   case OPT_USER_RM:
     ret = user.remove(user_op, &err_msg);
     if (ret == -ENOENT) {
-      cerr << err_msg << std::endl;
+      cerr << "could not remove user: " << err_msg << std::endl;
+      return -ret;
```

Comment:
can we just remove the special case for `-ENOENT` here, and let `if (ret < 0)` handle this below?